src/User.php

```diff
@@ -4851,12 +4851,12 @@ public function showItems($tech)
 
         $iterator = $DB->request([
             'SELECT'    => [
-                'glpi_groups_users.groups_id',
+                'glpi_groups.id',
                 'glpi_groups.name'
             ],
-            'FROM'      => 'glpi_groups_users',
-            'LEFT JOIN' => [
-                'glpi_groups' => [
+            'FROM'      => 'glpi_groups',
+            'JOIN' => [
```

Comment:
Is identical, but will prevent any ambiguity when reading the code.
```suggestion
            'LEFT JOIN' => [
```